src/status/contacts.nim

```diff
@@ -0,0 +1,39 @@
+import eventemitter
+import json
+import libstatus/contacts as status_contacts
+import profile
+
+type
+  Contact* = ref object
+    name*, address*, identicon*: string
+
+proc toContact*(self: JsonNode): Contact =
+  result = Contact(
+    name: self["alias"].getStr,
+    address: self["id"].getStr,
+    identicon: self["identicon"].getStr
+  )
```

Comment:
We're having `Profile` and `Contact` types. Might make sense to unify those?

Reply:
good point, will do this